reference/build_helpers/cmake.rst

```diff
@@ -65,10 +66,23 @@ Parameters:
     - **set_cmake_flags** (Optional, Defaulted to ``None``): Whether or not to set CMake flags like ``CMAKE_CXX_FLAGS``, ``CMAKE_C_FLAGS``, etc.
     - **msbuild_verbosity** (Optional, Defaulted to ``None``): verbosity level for MSBuild (in case of Visual Studio generator).
     - **cmake_program** (Optional, Defaulted to ``None``): Path to the custom cmake executable.
+    - **generator_platform** (Optional, Defaulted to ``None``): Generator platform name or none to autodetect (-A cmake option).
 
 Attributes
 ----------
 
+generator
++++++++++
+
+Specify a custom CMake generator to use, see also `cmake-generators documentation <https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html>`.
+If used before the ``configure()`` call, overrides the generator to use. If used after the ``configure()`` call, reports the actual generator in use (e.g. auto-detected one).
```

Comment:
Not sure what you mean by "reports the actual generator". Is there a difference if you read this variable before and after the configure() call?

Reply:
oh, you're right, auto-detection is done in ctor, not in configure, I'll change it